src/components/useList/components/ListItemExpandIcon/__stories__/list-item-expand-icon.md

```diff
@@ -0,0 +1,47 @@
+### ListItemExpandIcon
+
+Base group expand icon view
+
+#### Import
+
+```tsx
+import {
+  type unstable_ListItemExpandIconProps as ListItemExpandIconProps,
+  unstable_ListItemExpandIcon as ListItemExpandIcon,
+} from '@gravity-ui/uikit/unstable';
+```
+
+#### Base example:
+
+```jsx
+const DefaultExample = (props: ListItemExpandIconProps) => {
+    return <ListItemExpandIcon {...props} />;
+};
+```
+
+<ListItemExpandIconDefault />
+
+#### Render icon inside Button component:
+
+```jsx
+const InsideButtonExample = (props: ListItemExpandIconProps) => {
+    const [expanded, setExpanded] = React.useState(false);
+
+    return (
+        <Button onClick={() => setExpanded((x) => !x)}>
+            <ListItemExpandIcon {...props} expanded={expanded} />
+        </Button>
+    );
+};
+```
+
+<ListItemExpandIconInsideButton />
+
+#### Props
+
+| Name              | Description                                     |                         Type                         | Default |
+| :---------------- | :---------------------------------------------- | :--------------------------------------------------: | :-----: |
+| expanded          | icon state                                      |                      `boolean`                       |         |
+| disableTransition | disable animation while `expanded` state change |                      `boolean`                       |         |
```

Comment:
Why we need to disable the animatiom?

Reply:
It's our design decision to have transitions on such arrows. Look at the `ArrowToggle`, we don't have it. Allowing to disable it here would feels like something is broken...